### ISSUE TITLE
Is this project being maintained?

### ISSUE THREAD

**Issue by HacKanCuBa** [stackless-dev/stackless#306]
Hi all! I was wondering if this project was still a thing, or what, because I don't see any movement since the middle of last year...

**Comment by kristjanvalur** [comment]
Hi there.
I'm afraid that it has seen diminishing activity for the last few years.  @akruis has been the most active maintainer.  Merging stackless functionality into new versions of Python is always a big task.  3.11 looks to have enormous changes internally so it may not happen ever.

**Comment by maxrossi** [comment]
no news about 3.13?

**Comment by HacKanCuBa** [comment]
> no news about 3.13?

ehhh there hasn't been a single release since 3.8 :P

I'm pretty sure that this project is dead :(

**Comment by kristjanvalur** [comment]
Yes, I guess it is safe to say so :(.  We had a good run, though.
